locksmith/src/operations/verifierOperations.ts

```diff
@@ -0,0 +1,88 @@
+import { Verifier } from '../models/verifier'
+
+namespace VerifierOperations {
+  /**
+   * Returns true if the verifier already exists on the lock and network
+   * @param lockAddress
+   * @param address
+   * @param network
+   * @returns boolean
+   */
+  export const isVerifierAlreadyExits = async (
+    lockAddress: string,
+    address: string,
+    network: number
+  ): Promise<boolean> => {
+    return !!(await Verifier.findOne({
+      where: {
+        address,
+        lockAddress,
+        network,
+      },
+    }))
```

Comment:
separate into a variable and then do `!!`.

I would also just use `Boolean` which is more clear.

Reply:
rahaaa just saw this. Ok I will use `Boolean` in the future.